tests/test_miscellaneous_op.py

```diff
@@ -30,8 +29,8 @@ def _too_big_error(max_length: int) -> str:
 
 
 @pytest.fixture(scope="module")
-def get_ops_avm_result(algod_client: AlgodClient) -> AVMInvoker:
-    return create_avm_invoker(APP_SPEC, algod_client)
+def get_ops_avm_result(algorand: algokit_utils.AlgorandClient) -> AVMInvoker:
```

Comment:
```suggestion
def get_ops_avm_result(algorand: AlgorandClient) -> AVMInvoker:
```

Nitpick: for consistency.